include/pisa/accumulator/simple_accumulator.hpp

```diff
@@ -1,23 +1,26 @@
 #pragma once
-#include <vector>
-#include <cstddef>
-#include <algorithm>
 
-#include "topk_queue.hpp"
+#include <cstddef>
+#include <cstdint>
+#include <vector>
 
 namespace pisa {
 
-struct Simple_Accumulator : public std::vector<float> {
-    Simple_Accumulator(std::ptrdiff_t size) : std::vector<float>(size) {}
-    void init() { std::fill(begin(), end(), 0.0); }
-    void accumulate(uint32_t doc, float score) { operator[](doc) += score; }
-    void aggregate(topk_queue &topk) {
-        uint64_t docid = 0u;
-        std::for_each(begin(), end(), [&](auto score) { 
-            if(topk.would_enter(score)) { topk.insert(score, docid); } 
-            docid += 1;
-        });
-    }
+struct TopKQueue;
```

Comment:
Why this? `topk_queue.hpp` is pretty small overall

Reply:
Because we don't need it, so there's no reason to parse it at this point. I agree that in this particular situation we won't get much, but these things compound. But most importantly, our imports tend to be unnecessarily bloated, so reducing them is not a bad idea.

And most importantly, why not? Are we gaining anything by adding the include?